sympy/logic/boolalg.py

```diff
@@ -404,7 +402,6 @@ class Or(LatticeOp, BooleanFunction):
 
     >>> Or(x, y).subs(x, 0)
     y
```

Comment:
As a personal taste, I always prefer having a new line here. But I am neutral towards it.